### ISSUE TITLE
CAS is not supported by the client

### ISSUE THREAD

**Issue by RiccardoBravin** [yuezk/GlobalProtect-openconnect#368]
I'm trying to connect to a PaloAlto VPN which I was able to do before updating to Ubuntu 24.04 through your program. Now that i've reinstalled the system i cannot anymore because it reports:
`Error: Portal prelogin error: Prelogin failed: CAS is not supported by the client. Minimum client version is 6.0
`
Usually it immediately prompted me for the microsoft login screen.
I've also seen that the issue was once already solved but I imagine somehow the fix got lost?

**Logs**
```
[2024-06-01T10:09:50Z INFO  gpclient::cli] gpclient started: 2.1.2 (2024-03-29)
[2024-06-01T10:09:50Z INFO  gpapi::portal::prelogin] Prelogin with user_agent: PAN GlobalProtect
[2024-06-01T10:09:51Z INFO  gpclient::connect] Failed to connect portal with prelogin: Portal prelogin error: Prelogin failed: CAS is not supported by the client. Minimum client version is 6.0
[2024-06-01T10:09:51Z INFO  gpclient::connect] Trying the gateway authentication workflow...
[2024-06-01T10:09:51Z INFO  gpclient::connect] Performing the gateway authentication...
[2024-06-01T10:09:51Z INFO  gpapi::portal::prelogin] Prelogin with user_agent: PAN GlobalProtect

```
**Environment:**
 - OS: Ubuntu 24.04
 - Desktop Environment: GNOME

**Comment by yuezk** [comment]
Hi @RiccardoBravin CAS support was added in [2.1.3](https://github.com/yuezk/GlobalProtect-openconnect/releases/tag/v2.1.3), while your version is 2.1.2. Can you help upgrade to the latest version and try again? Thanks.

**Comment by RiccardoBravin** [comment]
Hey, @yuezk thank you very much. I didn't notice that for some reason newer versions were available and now that I've installed 2.3.1 everything works fine. Just a quick note, if you use the commands it somehow defaults to installing V 2.1.2 at least on my system, I don't know if that is the expected behavior.
Anyways thanks again :)

**Comment by yuezk** [comment]
Glad it works.

This could be cause I can no longer build the latest code on Launchpad for Ubuntu 24.04. The last build for 24.04 on Launchpad could be 2.1.2.

Since it works now, I'm closing this issue.
